src/App.js

```diff
@@ -1,25 +1,66 @@
 import React from 'react';
+import { Clock } from './components/Clock';
 
 import './App.scss';
 
-const App = () => {
-  setInterval(() => {
-    const date = new Date();
-
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
-};
-
-export default App;
+export class App extends React.Component {
+  state = {
+    isClockVisible: true,
+    clockName: 0,
+  };
+
+  lastState = this.state;
```

Comment:
don't save the last state here, use componentDidUpdate inside of a clock component, and just check if a name in prevState is the same as in the state